### ISSUE TITLE
Make Gravatar optional and not the default

### ISSUE THREAD

**Issue by evrifaessa** [FOSSBilling/FOSSBilling#176]
We should keep our dependence to external services at minimum. Gravatar should be optional.

**Comment by jaapmarcus** [comment]
I agree 

Same like the Discord "box" in the installer and so on...

**Comment by evrifaessa** [comment]
Yeah, I guess we should turn this thread into a list of our external dependencies.

**Comment by evrifaessa** [comment]
Gravatar should become a standalone module. We should offer more extensibility for the profile picture.

**Comment by John-S4** [comment]
> Gravatar should become a standalone module. We should offer more extensibility for the profile picture.

I'm not sure it needs to be s separate module, I think it is fine to have it in there by default, just switched off and with an alternative option to upload profile pics without using it.

**Comment by admdly** [comment]
@evrifaessa looks like all external font dependencies have now been removed with #840, might be work marking that task complete?
